extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DevServicesBuildTimeConfig.java

```diff
@@ -83,11 +84,11 @@ public interface DevServicesBuildTimeConfig {
     Optional<String> password();
 
     /**
-     * The path to a SQL script to be loaded from the classpath and applied to the Dev Service database.
+     * The paths to SQL scripts to be loaded from the classpath and applied to the Dev Service database.
      * <p>
      * This has no effect if the provider is not a container-based database, such as H2 or Derby.
      */
-    Optional<String> initScriptPath();
+    Optional<List<@WithConverter(TrimmedStringConverter.class) String>> initScriptsPaths();
```

Comment:
Better keep the name `initScriptPath` -- otherwise you'll break existing applications that leverage this setting for little reason.

We *could* rename it if really necessary, but that would require taking some steps to help people migrate, which is work, and not worth it here IMO.